Assets/Code/Scripts/LevelManagement/PlayerSpawner.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class PlayerSpawner: MonoBehaviour
+{
+    private GameManager _gameManager;
+    
+    [SerializeField] private Transform player;
+    
+    /*
+     *  PlayerSpawner handles where Player should spawn after hitting certain objects.
+     *  For now, when the Player collides with a trigger object, he respawns in the last saved checkpoint.
+     *  Note: checkpoints for now are volatile, meaning that closing the game will erase them.
+     */
+    
+    void Start()
+    {
+        _gameManager = GameObject.FindGameObjectWithTag("GameManager").GetComponent<GameManager>();
```

Comment:
Could this line be in an Awake function?

Reply:
I'll have a look.